routes/board_routes.rb

```diff
@@ -81,6 +81,7 @@ class Ollert
 
     @board_id = board_id
 
+    @title = "#{@board_name} Board"
```

Comment:
I was on the fence. I'll nuke it.
On Mar 23, 2015 7:25 AM, "Larry Price" <notifications@github.com> wrote:

> In routes/board_routes.rb
> <https://github.com/sep/ollert/pull/15#discussion_r26928668>:
>
> > @@ -81,6 +81,7 @@ class Ollert
> >
> >      @board_id = board_id
> >
> > +    @title = "#{@board_name} Board"
>
> I don't think I see the need for the "Board" suffix - your thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sep/ollert/pull/15/files#r26928668>.
>